src/code2vec.py

```diff
@@ -39,6 +43,10 @@ def main():
                         required=False)
     parser.add_argument('-w', '--max_width', type=int, default=2, help="Max path width.",
```

Comment:
create subparser, let's compartmentalize data and ml pipeline -> I'd suggest commands should be `extract_features` and then `train_model` (for now only add the first and a TODO for the second, if you have better name pls propose)

Reply:
not sure about this bit. should I create two methods like `add_extract_features(parser)` and `add_train_model_args(parser)` and add the respective options inside of them?

Reply:
i think for one you can omit the second one, as no logic for it exists. you can do something like:

```
subparsers = parser.add_subparsers(help="Commands", dest="command")
extract_parser = subparsers.add_parser(
            "extract", help="Extract features from input repositories", formatter_class=args.ArgumentDefaultsHelpFormatterNoNone)
```

Reply:
I do not know how to do this, I've pushed my trial (which does not work) but I'm not sure how these hierarchies of parser work, the rest should be solved.